tests/conftest.py

```diff
@@ -1,12 +1,16 @@
 """Collection of fixtures to assist the testing scripts."""
+from logging import DEBUG
 from typing import Any
 
 import pytest
 from xarray import DataArray
 
 # An import of LOGGER is required for INFO logging events to be visible to tests
 # This can be removed as soon as a script that imports logger is imported
-import virtual_rainforest.core.logger  # noqa
+from virtual_rainforest.core.logger import LOGGER
+
+# Class uses DEBUG
+LOGGER.setLevel(DEBUG)
```

Comment:
I'm not sure about this... I really don't think you need to check that log statements at the debug level are emitted. The debug level is aimed at developers only so no one should be relying on its output. If you really want everyone to see it, use the info level and then it makes more sense to test it.

Reply:
Tbf, I've just realised that this was the existing behaviour and that you've just moved things around. I don't think you guys want to check for debug logging though. Maybe open an issue/discussion?